omero/sysadmins/server-upgrade.rst

```diff
@@ -272,7 +272,7 @@ to run.
     (1 row)
 
 
-If you are upgrading from a server earlier than |previousversion| then
+If you are upgrading from a server earlier than 5.3, then
```

Comment:
Will be staged at https://merge-ci.openmicroscopy.org/jenkins/job/OMERO-docs/lastSuccessfulBuild/artifact/omero/_build/html/sysadmins/server-upgrade.html#run-the-upgrade-script once job is green.